lib/index.js

```diff
@@ -487,6 +487,9 @@ module.exports = (markdown, options) => {
     options.marked.renderer = new ConfluenceRenderer(options);
 
     // Replace "\r\n" and "\r" with "\n".
+		// Convert Buffer to String for replacement to work
```

Comment:
Would you ... ?
1. Move the code up above the "Replace" comment
2. Remove the console log
3. Fix the indentation
4. Ensure the spacing matches the rest of the file